docker/.env

```diff
@@ -22,7 +22,8 @@ ES_DISCOVERY_TYPE=single-node
 # --------------------------------------------------------------------
 
 # Docker image of the Registry API
-REG_API_IMAGE=nasapds/registry-api-service:1.1.10
+#REG_API_IMAGE=nasapds/registry-api-service:1.1.10
+REG_API_IMAGE=nasapds/registry-api-service:latest
```

Comment:
@tloubrieu-jpl , Is this change intentional? I understand that we had some issues with some versions of Registry API. However, is it possible to have an exact working Registry-API version here instead of latest version?

Reply:
@ramesh-maddegoda yes that was intentional, for the main branch HEAD I was thinking to have the latest images and for the tagged version the tagged images. At least that is how I envisioned the usage of this docker compose.

To have the latest images is useful for development/integration testing purpose.

When I'll do a tag for registry 1.1.3, later today, I'll switch to tagged version for all the images.